packages/react/src/patterns/sections/CardSection/__stories__/CardSection.stories.js

```diff
@@ -34,12 +40,20 @@ storiesOf('Patterns (Sections)|CardSection', module)
 
     cards[type] = data;
 
+    const toggleCTA = boolean('cta', true);
+    const cta = {
+      heading: 'Top level card link',
+      cta: {
+        href: 'https://www.example.com',
+      },
+    };
+
     return (
       <CardSection
         heading={text('Heading (required)', cardsTitle)}
```

Comment:
Small thing, but if you move this somewhere above line 39 I think this will be one of the first things instead of one of the last.

Reply:
updated @photodow